routers/web/repo/pull_review.go

```diff
@@ -242,3 +245,50 @@ func DismissReview(ctx *context.Context) {
 
 	ctx.Redirect(fmt.Sprintf("%s/pulls/%d#%s", ctx.Repo.RepoLink, comm.Issue.Index, comm.HashTag()))
 }
+
+const headCommitKey = "_headCommitSHA"
+
+func UpdateViewedFiles(ctx *context.Context) {
+	issue := checkPullInfo(ctx)
+	if ctx.Written() {
+		return
+	}
+	pull := issue.PullRequest
+	updatedFiles := make(map[string]pulls.ViewedState, len(ctx.Req.Form))
+	headCommitSHA := ""
+
+	// Collect all files and their viewed state
+	for file, values := range ctx.Req.Form {
+		for _, viewedString := range values {
+			viewed, err := strconv.ParseBool(viewedString)
```

Comment:
Could we have a prefix for these keyes?

Reply:
For every value that is a boolean, we can assume that it is a file.
Or do you plan on sending other booleans with the request? That would be the only case where a prefix is necessary. However, the route in its current form is not intended to take non-file boolean arguments.

And even if we implement the prefix, what kind of prefix do you suggest?